package.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "payment-sdk",
+  "author": "",
+  "license": "ISC",
+  "version": "1.0.0",
+  "description": "",
+  "homepage": "https://github.com/XcooBee/payment-sdk-js#readme",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/XcooBee/payment-sdk-js.git"
+  },
+  "bugs": {
+    "url": "https://github.com/XcooBee/payment-sdk-js/issues"
+  },
+  "main": "dist/node/node.js",
+  "module": "dist/node/node.js",
+  "browser": "dist/browser/browser.js",
+  "scripts": {
+    "test": "mocha -r ts-node/register tests/**/*.test.ts",
+    "coverage": "nyc -r lcov -e .ts -x \"*.test.ts\" npm run test",
```

Comment:
Do we have anything that will automatically run the coverage reports? For a recent project, I've set up Circle CI to run format checks, lint, and run tests with coverage reporting that only succeeds if at 100% for four different categories. Since this project is open source, we can use Circle CI for free (if I remember correctly).

Is there anything to enforce it stays at 100%?